docs/docs/api/Tasks/fetchSingleTask.md

```diff
@@ -7,68 +7,68 @@ borderRadius: '2px',
 color: '#E83E8C',
 }}>{children}</span> );
 
-<Endpoint>GET tasks/task/:id/</Endpoint>: Fetching a single Task
+<Endpoint>GET /tasks/task/:id</Endpoint>: Fetching a single Task
 
 ```json
-{
-
-}
+{}
 ```
 
 ### Example Request
+
 This is a **protected route**, a **valid JWT is required** in the header field
+
 #### Header
+
 ```
 Authorization: Bearer eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJleHAiOjE1OTU4MjQyNzUsImlhdCI6IjIwMjAtMDctMjdUMDA6MjY6MTUuNzg5NTg0Mi0wNDowMCIsInN1YiI6ImNocmlzIn0.5US2_ITKcfgkpEbfsR-gxXbGPFY6XsgJPcGA5qaBD1M
 ```
 
 #### Body
-```json
-{
 
-}
+```json
+{}
 ```
 
 #### Parameters
-```json
-{
-    id: task_id // Id of the task that is being searched for
-}
 
-```
+    id: task_id // Id of the task that is being searched for
 
 ### Possible Responses
+
 #### Immediate Success
+
 ```json
 {
-	"code": 500,
-	"message": "Could not fetch doc",
-	"data": {}
+  "code": 200,
```

Comment:
Is this truly the return that we're sending to users? This doesn't seem like the format of the return that we're sending. (I'm pretty sure we don't even return a code within the returned body)

Reply:
I would double check with the body that you're getting as a return and update it accordingly (same with the other documents as well)

Reply:
![image](https://user-images.githubusercontent.com/24628243/160250684-edc18ea2-b793-410d-b740-65a65a734fc0.png)
For example, this is what we truly return. (For the portion that says `stack`, I wouldn't include the entire error message since it's unnecessary to)

Please make sure that you change what I provided as a template to the proper return body that we're returning